### PR TITLE
Allow session lifetime to be overridden from environment/dotenv

### PR DESCRIPTION
**Pull request by JamieDriver**

### CODE REVIEW

File: flaskserver.py [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
 b2h = hex_from_bytes
 h2b = hex_to_bytes
 
 # Time we will retain active sessions, in seconds.
 # ie. maximum time allowed 'start_handshake' (which creates the session)
 # and the get-/set-pin call, which utilises it.
-SESSION_LIFETIME = 30
+# Can be set in environment, defaults to 30s
+load_dotenv()
+SESSION_LIFETIME = os.environ.get(int(os.environ.get('SESSION_LIFETIME', 30)))
```
Comment (jgriffiths) [inline flaskserver.py:19]: I think you have an extra call to `os.environ.get` here? Only the inner `int()` expression seems needed...

Comment (JamieDriver) [reply flaskserver.py:19]: yy daft-o

**Comment by JamieDriver** [comment]
As discussed, set the default to 5mins, and have the test use the env to set a short lifetime.

**Comment by jgriffiths** [comment]
utack 188fdb01797074de92fb4ab2deaeeb697c44ef85

**Comment by greenaddress** [comment]
utack 188fdb01797074de92fb4ab2deaeeb697c44ef85

**Pull request merged**